Smach_image_publisher shows error during kitchen demo.

`smach_image_publisher` shows error during kitchen demo.

```
[ERROR] [1687845042.887504] [/smach_image_publisher:rosout]: bad callback: <bound method SmachImagePublisher._status_msg_update of <__main__.SmachImagePublisher object at 0x7f05a4f9a790>>
Traceback (most recent call last):                                                   
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback                
    cb(msg)                                                                               
  File "/home/fetch/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/src/smach_viewer/smach_viewer_base.py", line 489, in _status_msg_update
    if container.update_status(msg):                                                                                                     
  File "/home/fetch/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/src/smach_viewer/smach_viewer_base.py", line 101, in update_status
    self._local_data._data = self._load_local_data(msg)                                                             
  File "/home/fetch/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/src/smach_viewer/smach_viewer_base.py", line 75, in _load_local_data
    local_data = pickle.loads(msg.local_data.encode('utf-8'))                        
UnicodeDecodeError: 'ascii' codec can't decode byte 0x88 in position 11: ordinal not in range(128)    
```

--- a/jsk_fetch_robot/jsk_fetch.rosinstall.melodic
+++ b/jsk_fetch_robot/jsk_fetch.rosinstall.melodic
@@ -151,10 +151,16 @@
     uri: https://github.com/jsk-ros-pkg/coral_usb_ros.git
     version: master
 # we need melodic-devel branch for headless visualization
+# - git:
+#     local-name: ros-visualization/executive_smach_visualization
+#     uri: https://github.com/ros-visualization/executive_smach_visualization.git
+#     version: master
+# UnicodeDecodeError is output without the following PR
+# https://github.com/ros-visualization/executive_smach_visualization/pull/54
 - git:
     local-name: ros-visualization/executive_smach_visualization
-    uri: https://github.com/ros-visualization/executive_smach_visualization.git
-    version: melodic-devel
+    uri: https://github.com/tkmtnt7000/executive_smach_visualization.git
+    version: fix-pickle-loads-python-version
 # we need this for eus10 and roseus_resume
 - git:
     local-name: euslisp/Euslisp

